### PR TITLE
#438 Improve Ohara README.md and Manager README.md

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] Set a Milestone (we usually file PR for master branch)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

required by #438

### CODE REVIEW

File: ohara-manager/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Make sure you're at the Ohara manager root, then start it with:
 yarn start --configurator http://host:port/v0
 ```
 
-> Note that the configurator option is required, and you should have configurator running before starting Ohara manager.
+> Note that the configurator option is required, and you should have [configurator](https://github.com/oharastream/ohara/blob/master/README.md) running before starting Ohara manager.
```
Comment (konekoya) [inline ohara-manager/README.md]: Let's change this line to:
> Note that the configurartor option is required, and you should have configurator running before starting Ohara manager. You can see the main [README.md](https://github.com/oharastream/ohara/blob/master/README.md#running-configurator-by-docker) on how to spin up a configurator

Comment (wu87988622) [reply ohara-manager/README.md]: Your suggestion is wonderful!!!!

**Comment by chia7712** [comment]
Please review Review Contribution Guidelines and then click the checkbox :)

File: ohara-manager/README.md (hunk 1 above)
Comment (chia7712) [reply ohara-manager/README.md]: Please "resolve conversation" if you have addressed @konekoya's comment

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Ohara
 
+Test this
```
Comment (chia7712) [inline README.md]: hello world

File: ohara-manager/README.md (hunk 1 above)
Comment (konekoya) [reply ohara-manager/README.md]: Yep, you can resolve this conversation now. Thank you :)

File: README.md (hunk 2 above)
Comment (wu87988622) [reply README.md]: Is fix

File: README.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ gradle licenseApply
 **Running Ohara In Docker**
 
 - Ensure your nodes (actual machines or VMs) have installed Docker 18.09+
+  - You con see the main [README.md](https://github.com/oharastream/ohara/blob/master/docker/README.md) on how to spin up a Docker
```
Comment (konekoya) [inline README.md]: I'd rephrase this to something like:
_Ensure your nodes (actual machines or VMs) have installed Docker 18.09 or up. For Docker installation instructions you can see it [here](https://github.com/oharastream/ohara/blob/master/docker/README.md)_
